sdgym/synthesizers/generate.py

```diff
@@ -1,95 +1,63 @@
-"""Synthesizers module."""
+"""Helpers to create SDV synthesizer variants."""
```

Comment:
If this module is `generate.py` then either this description is wrong (because we have RealTabFormer in here too) or the functions you added don't belong here.

I believe this is `Helpers to create SDGym synthesizer variants`, that may be more appropriate.